### PR TITLE
Clone sap-styleguides repo

### PR DESCRIPTION
**Pull request by melissaxd96**
- clone sap-styleguides repo

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# global owners as fallback for anything not matched below
```
Comment (jcailan) [inline CODEOWNERS]: just replace everything with

`* @jcailan`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,90 @@
 # best-practices
 Rizing Innovation Best Practices
```
Comment (jcailan) [inline README.md]: Just remove the first 2 lines for now

**Review by jcailan — CHANGES_REQUESTED** [review]
PR description -- this is not an sbb project.

File: clean-abap/RAP.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
+## Content
```
Comment (jcailan) [inline clean-abap/RAP.md]: Remove `RAP.md` since it needs rework.

**Review by jcailan — CHANGES_REQUESTED** [review]

File: clean-abap/RAP.md (hunk 3 above)
Comment (melissaxd96) [reply clean-abap/RAP.md]: this is done

**Review by jcailan — DISMISSED** [review]

**Review by jcailan — APPROVED** [review]

**Pull request closed without merging**